server/testutils/factories/person.ts

```diff
@@ -2,10 +2,12 @@ import { Factory } from 'fishery'
 import { faker } from '@faker-js/faker/locale/en_GB'
 
 import type { FullPerson, PersonSummary, RestrictedPerson } from '@approved-premises/api'
+import { FullPersonSummary } from '@approved-premises/api'
 import { DateFormats } from '../../utils/dateUtils'
 
+export const getCrn = () => `C${faker.number.int({ min: 100000, max: 999999 })}`
```

Comment:
Could you rename this to `createCRN`, as that is slightly more descriptive of what it is doing